presets/ras-defaults.json

```diff
@@ -0,0 +1,334 @@
+                    "category",
+                    "tag",
+                    "author",
+                    "date",
+                    "post-type",
+                    "taxonomy"
+                ],
+                "additional_classes": "",
+                "excluded_categories": [],
+                "excluded_tags": []
+            },
+            "categories": [],
+            "tags": false,
+            "campaign_groups": [
+                {
+                    "id": 30,
+                    "name": "Reader Activation Defaults"
+                }
+            ]
+        }
+    ],
+    "segments": [
+        {
+            "name": "Registered, signed up for at least one newsletter, but has not donated",
+            "configuration": {
+                "min_posts": 0,
+                "max_posts": 0,
+                "min_session_posts": 0,
+                "max_session_posts": 0,
+                "is_subscribed": true,
+                "is_not_subscribed": false,
+                "is_donor": false,
+                "is_not_donor": true,
+                "is_former_donor": false,
+                "is_logged_in": true,
+                "is_not_logged_in": false,
+                "favorite_categories": [],
+                "referrers": "",
+                "referrers_not": ""
+            },
+            "id": "639b3315239f0",
+            "priority": 0
+        },
+        {
+            "name": "Registered but not signed up for a newsletter",
+            "configuration": {
+                "min_posts": 0,
+                "max_posts": 0,
+                "min_session_posts": 0,
+                "max_session_posts": 0,
+                "is_subscribed": false,
+                "is_not_subscribed": true,
+                "is_donor": false,
+                "is_not_donor": false,
+                "is_former_donor": false,
+                "is_logged_in": true,
+                "is_not_logged_in": false,
+                "favorite_categories": [],
+                "referrers": "",
+                "referrers_not": ""
+            },
+            "id": "639b333b41025",
+            "priority": 1
+        },
+        {
+            "name": "Not registered and not signed up for a newsletter",
+            "configuration": {
+                "min_posts": 0,
+                "max_posts": 0,
+                "min_session_posts": 0,
+                "max_session_posts": 0,
+                "is_subscribed": false,
+                "is_not_subscribed": true,
+                "is_donor": false,
+                "is_not_donor": false,
+                "is_former_donor": false,
+                "is_logged_in": false,
+                "is_not_logged_in": true,
+                "favorite_categories": [],
+                "referrers": "",
+                "referrers_not": ""
+            },
+            "id": "639b337fdbf33",
+            "priority": 2
+        }
+    ],
+    "campaigns": [
+        {
+            "id": 30,
```

Comment:
Will the importer handle a possible campaign ID conflict?

Reply:
yes, the id only serves the purpose of creating the relation between prompts and campaigns, it's not used when creating the term in the destination site